test/cpp/naming/resolver_component_tests_runner_invoker_for_bazel.cc

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ * Copyright 2017 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/* Intended for running the resolver component test under run_tests.py */
```

Comment:
Instead of having two different binaries for this, can we have a single binary that just constructs the paths differently based on the flags that are passed in?  It seems like the code would be less spread out that way.

Reply:
done, yeah that got rid of a lot